docs/source/index.rst

```diff
@@ -13,6 +13,7 @@ Contents:
 
    overview
    install
+   user_guide
```

Comment:
PR #195 just got merged.  If you rebase your PR onto master, the build on Travis CI will catch that.